packages/utilities/src/rtl.ts

```diff
@@ -61,3 +61,15 @@ export function getRTLSafeKeyCode(key: number): number {
 
   return key;
 }
+
+function _clearRTL(): void {
+  _isRTL = undefined;
+}
+
+/**
+ * @internal Unit test interface.
+ */
+// tslint:disable-next-line:variable-name
+export const __testHooks = {
+  _clearRTL
+};
```

Comment:
How do people feel about this approach? I've wanted to do this a few times now and we don't have a convention that I know of. I think it's good to have unit tests for modules like these but the static variables require a way of letting unit tests reset state.

Reply:
This also impacts bundle size a bit, which is concerning, but I feel like we need some way to get better test coverage for these types of modules. (Unless we can eliminate this from bundle size somehow.)